application/testing/auto_reload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
should we use a Python script instead of bash to have a single script for all OS?

Reply:
i'd rather write a powershell script instead of adding a dependency to python in our testing.